oak/client/application_client.h

```diff
@@ -28,6 +28,7 @@
 #include "oak/common/logging.h"
```

Comment:
Are the asylo includes further up (20-21) still needed? (If not, there's probably a BUILD dep that can go too)

Reply:
Done.